test/utils/SecureCommsMessageTestData.scala

```diff
@@ -463,6 +463,24 @@ object SecureCommsMessageTestData {
       PreferencesModel(EMAIL, DIGITAL, ENGLISH, TEXT)
     )
 
+    val messageModelDeRegistrationInvalidTemplateNoFields = SecureCommsMessageModel(
```

Comment:
Confusing naming - this model has `None` for all the fields that would define it as a particular change, but it's named as a deregistration model? If it was a dereg model it would have a dereg date.